evaluation.py

```diff
@@ -0,0 +1,101 @@
+from submodules.HeatMap import HeatMap # https://github.com/LinShanify/HeatMap
+from constants import *
+# from util import *
+
+import matplotlib.pyplot as plt
+import numpy as np
+from PIL import Image
+import cv2
+
+class Evaluation():
+
+    def __init__(self, model, weights, df, num_hg_blocks, batch_size=1):
+        self.model = model              # json of model to be evaluated
```

Comment:
nit: calling this `model` is a bit confusing. Is it the `model_arch` or the model class itself (aka an instance of Hourglass class)?